src/Dialogs/Preferences.vala

```diff
@@ -42,4 +49,58 @@ public class Tooth.Dialogs.Preferences : Adw.PreferencesWindow {
         settings.bind ("show-spoilers", show_spoilers, "active", SettingsBindFlags.DEFAULT);
 	}
 
+	[GtkCallback]
+	private void on_scheme_changed () {
+		var selected_item = (ColorSchemeListItem) scheme_combo_row.selected_item;
+		var style_manager = Adw.StyleManager.get_default ();
+
+		style_manager.color_scheme = selected_item.adwaita_scheme;
+		settings.color_scheme = selected_item.color_scheme;
+	}
+}
+
+public class Tooth.ColorSchemeListModel : Object, ListModel {
```

Comment:
Both ColorSchemeListModel and ColorSchemeListItem are requisites to use Adw.ComboRow, as we need an item and a list model. I preferred to use this implementation as I felt using Gtk.StringList and then converting the value to an enum and setting it would be a little 

The ColorSchemeListItem has two properties, name and adwaita_scheme, which are used to display the name of the enumeration to the Adw.ComboRow, and adwaita_scheme to set the value to the style_manager when it is obtained.